runtime/doc/lsp.txt

```diff
@@ -1130,23 +1130,23 @@ code_action({options})                             *vim.lsp.buf.code_action()*
     Selects a code action available at the current cursor position.
 
     Parameters: ~
-      • {options}  (table|nil) Optional table which holds the following
-                   optional fields:
-                   • context: (table|nil) Corresponds to `CodeActionContext` of the LSP specification:
-                     • diagnostics (table|nil): LSP `Diagnostic[]`. Inferred
-                       from the current position if not provided.
-                     • only (table|nil): List of LSP `CodeActionKind`s used to
+      • {options}  (table?) Optional table which holds the following optional
+                   fields:
+                   • context?: table Corresponds to `CodeActionContext` of the LSP specification:
```

Comment:
I'll look into this one too.

Reply:
Questions for clairty:

- Do we want parentheses around the type? I think the parentheses improve readability. Parentheses usage in this file is inconsistent, so I could work on improving this is future PRs.

- Is this syntax correct? (e.g. `NAME: TYPE DESCRIPTION`) The syntax for the table entries looks more appropriate in my opinion (`NAME TYPE: DESCRIPTION`). I don't have enough contextual knowledge to know why they're different. But this could be another "consistency" task for the future.

Reply:
- Yes
- For Lua annotations? Both wrong. It should be `NAME TYPE DESCRIPTION`.
  The syntax is `---@param <name[?]> <type[|type...]> [description]` https://github.com/LuaLS/lua-language-server/wiki/Annotations#param

Reply:
There are many incorrect usages of this syntax. I think it is best to leave this work for another PR, since this would create a diff for many more lines.

Reply:
But it would be ncie to have parentheses around the type. I'll look into that.